src/raftstore/store/fsm/peer.rs

```diff
@@ -1472,6 +1477,9 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
                 let id = peer.get_id();
                 self.fsm.peer.peer_heartbeats.insert(id, now);
                 if self.fsm.peer.is_leader() {
+                    // Speed up snapshot instead of waiting another heartbeat.
+                    self.fsm.peer.ping();
```

Comment:
seems it's only needed by AddLearnerNode?

Reply:
Just in case learner is disabled.